src/reducers/tokenBalances.ts

```diff
@@ -7,16 +7,16 @@ import { setTokenBalance } from 'actions/tokenBalances'
 // const zeroBalance = codeList.reduce((acc, code) => (acc[code] = 0, acc), {}) as TokenBalances
 
 // TODO: fill in when we have actions + fill in rest of Tokens + proper typing
-export default handleActions<TokenBalances | any>(
+export default handleActions<TokenBalances>(
   {
     [setTokenBalance.toString()]: (state, action) => ({
       ...state,
-      [action.payload.tokenName]: action.payload.balance,
+      [action.payload['tokenName']]: action.payload['balance'],
```

Comment:
why? TS fault?

Reply:
yeah weird error saying `tokenName` non existant on `TokenBalances` type

Reply:
change to 
```
handleActions<TokenBalances, {tokenName: string, balance: string}>(
```

second temlate arg is payload